index.html

```diff
@@ -144,57 +128,84 @@ <h3>Multi-Post Stories Gain+Glory</h3>
+          <h2>About me</h2>
+          
+          <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>
 
-        <button type="submit" class="resume-button">Get my Resume</button>
-      </article>
+          <button type="submit" class="resume-button">Get my Resume</button>
+        </article>
 
         <article class="card-about-me">
-         <div class="icon-holder"><img src="imgs/lang-icon.png" alt="lang-icon" width="48" height="48"></div>
-         <h3>Languages</h3>
-         <ul>
-          <li>JavaScript</li>
-          <li>Ruby</li>
-          <li>Html</li>
-          <li>CSS</li>
-         </ul>
+          <div class="icon-holder"><img src="imgs/lang-icon.png" alt="lang-icon" width="48" height="48"></div>
+          <h3>Languages</h3>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
         </article>
        
         <article class="card-about-me">
           <div class="icon-holder"><img src="imgs/frameworks-icon.png" alt="frameworks-icon" width="48" height="48"></div>
-         <h3>Frameworks</h3>
-         <ul>
-          <li>Bootstrap</li>
-          <li>Ruby on rails</li>
-          <li>RSpec</li>
-          <li>Capybara</li>
-          <li>Selenium</li>
-         </ul>
+            <h3>Frameworks</h3>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
         </article>
         
         <article class="card-about-me">
-          <div></div><div class="icon-holder"><img src="imgs/skills-icon.png" alt="skills-icon" width="48" height="48"></div>
+          <div class="icon-holder"><img src="imgs/skills-icon.png" alt="skills-icon" width="48" height="48"></div>
           <h3>Skills</h3>
           <ul>
-           <li>Codekit</li>
-           <li>GitHub</li>
-           <li>Codepen</li>
-           <li>Gitlab</li>
-           <li>Terminal</li>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
           </ul>
         </article>
+      </section>
+
+      <section class="contact-section">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
 
+        <form action="https://formspree.io/f/xvolveaz" method="post">
+          
+          <ul>
+            <li><input type="text" name="first_name" placeholder="Full name" required maxlength="30"/></li>
+            <li><input type="email" name="email" placeholder="Email address" required/></li>
+            <li>
+            <textarea required maxlength="500">Enter text here</textarea>
```

Comment:
Kindly make the `Enter text here` `placeholder text` for a better user experience 🤗

Reply:
This is what the figma design specify

Reply:
- [ ] As the previous reviewer mentioned and according to the requirements all fields should have placeholder text, kindly put "Enter text here" inside the placeholder attribute 👌 
- [ ] And to match the Figma design you can style it by adding in your CSS file 💡
```
textarea::placeholder {
  color: #091e42;
  padding: 12px;
}
```

![image](https://user-images.githubusercontent.com/45231276/187041592-11e310f3-557c-41e7-be2c-d047b64822cc.png)